### ISSUE TITLE
Hitmap / scan strategy details

### ISSUE THREAD

**Issue by msyriac** [simonsobs/map_based_simulations#37]
The mapsims origin currently says:
"It also includes simulated relative hitmaps simulated in time domain with TOAST."

Could we add more information somewhere on these time domain simulations? Is there a version number and person associated with the run? (I was specifically interested in knowing the duration of the scan simulated).

**Comment by msyriac** [comment]
Here's the hitmap for LT3 in equatorial coordinates to help jog memories:
![2020-09-23_16:53:39](https://user-images.githubusercontent.com/3674397/94070573-72375b00-fdc0-11ea-9114-7006d3f6a500.png)

**Comment by keskitalo** [comment]
@zonca can you tell if you are using an individual hit map or if you combined the hits from the four provided splits?

**Comment by zonca** [comment]
thanks @msyriac, yes, this needs tracking.

@keskitalo I am using `LT0_UHF1_01_of_20.nominal_telescope_all_time_all_hmap.fits.gz`,  see: https://portal.nersc.gov/project/sobs/so_mapsims_data/v0.2/healpix/, see older discussion here: https://github.com/simonsobs/map_based_simulations/issues/21

What should I do instead? can you also point me to location of hitmaps? The old location is not available anymore.

**Comment by zonca** [comment]
@keskitalo was there a decision taken after the email exchange about this? if so can you summarize here?

**Comment by keskitalo** [comment]
The hitmaps accurately describe `mss-0001` so I don't think there is immediate need to update them. Sara is working to have a more definite answer on what the SO LAT scan strategy will be. Once there is consensus, the hitmap should reflect that.

**Comment by zonca** [comment]
thanks @keskitalo, for the time being should we replace the single split I used with a combination of all the 20 (right?) splits?

**Comment by keskitalo** [comment]
You can, if you want. The splits were engineered to provide almost uniform coverage.

**Comment by zonca** [comment]
thanks @keskitalo,
it looks like combining the splits wouldn't improve the situation much, I think it is not worth it.

I added a pointer to the MSS-0001 docs into the `mapsims` docs. and also to the last noise release https://github.com/simonsobs/map_based_simulations/commit/9de05ade7ac59bfc8959a18eb2406aed233455b9

let's wait for the new hitmaps to prepare another quick release.
@msyriac if you have other feedback please reopen
